### PR TITLE
Pm fixes

### LINKED ISSUE #54: TPPTokenConnection not able to customize OAuth scope
**PROBLEM SUMMARY**
TPPTokenConnection always uses the `certificate:manage,revoke;configuration:manage` scope when POSTing to /vedauth/authorize/oauth. This is hardcoded, as opposed to customizable.

On my TPP this causes an authorization issue. I'm not able to authorize with my TPP until I remove the 'configuration:manage' scope.

**COMMENTS/WORKAROUNDS**
TPPTokenConnection's constructor creates an Authentication object with no way to customize its `scope` property. As a result, the Authentication object always uses the default `SCOPE` constant, whose value is `certificate:manage,revoke;configuration:manage`.

### PR DESCRIPTION
**Pull request by rvelaVenafi**
Fixes #54 and #55 

Fixed an issue related with domain names in VaaS platform. A regex is added to them when creating a policy, but are not removed when reading that same policy. This issue effectively rendered the policy template unusable for future policy creation actions.

Fixed an issue on TPP policy management test suite. The random policy name was being generated incorrectly and made tests fail.

Fixed an issue on VaaS policy management logic. The Country regexes were being passed as the common name regexes due to an error in variable naming.

### CODE REVIEW

File: tests/test_pm.py [hunk 1]
```diff
@@ -164,6 +175,9 @@ def create_policy(connector, zone, policy_spec=None, policy=None, defaults=None)
     return resp
 
 
+POlICY_DOMAINS = ['vfidev.com', 'vfidev.net', 'venafi.example']
```
Comment (tr1ck3r) [inline tests/test_pm.py:178]: @rvelaVenafi is there a reason for the lower case "l" here? i.e. why is it not `POLICY_DOMAINS`?

Comment (rvelaVenafi) [reply tests/test_pm.py:178]: Of course there is! A big big typo!
Fixing it asap.

File: vcert/common.py [hunk 2]
```diff
@@ -556,12 +556,17 @@ def __init__(self, req_id=None, thumbprint=None,  reason=RevocationReasons.NoRea
 
 
 CLIENT_ID = "vcert-sdk"  # type: str
-SCOPE = "certificate:manage,revoke;configuration:manage"  # type: str
+# Certificate Management scope
+SCOPE_CM = "certificate:manage,revoke"  # type: str
+# Policy Management scope
+SCOPE_PM = 'certificate:manage;configuration:manage'  # type: str
+# Full Management scope
+SCOPE_FULL = 'certificate:manage,revoke;configuration:manage'  # type: str
```
Comment (tr1ck3r) [inline vcert/common.py:564]: Nit: inconsistent use of double/single quotes

File: tests/test_pm.py (hunk 1 above)
Comment (rvelaVenafi) [reply tests/test_pm.py]: Done!

File: vcert/common.py (hunk 2 above)
Comment (rvelaVenafi) [reply vcert/common.py]: Changed to double quotes as those are the ones commonly used for plain strings.

**Review by tr1ck3r — APPROVED** [review]
Thank you @rvelaVenafi, looks good!

**Review by angelmoo — APPROVED** [review]
changes looks good.

**Pull request merged**